test/org/sagebionetworks/bridge/services/ScheduledActivityServiceMockTest.java

```diff
@@ -665,6 +665,23 @@ public void getActivityHistoryV3ValidatesDateRange() {
         service.getActivityHistory(HEALTH_CODE, ActivityType.SURVEY, "GUID", endsOn, startsOn, "offsetKey", 20);
     }
     
+    @Test
+    public void oneTimeActivityDisappearsOutOfTimeRange() throws Exception {
+        DateTime endsOn = DateTime.now().plusDays(3);
+        ScheduleContext context = createScheduleContext(endsOn).build();
+        
+        mockAllCallsForDbActivities(Lists.newArrayList());
+        
+        service.getScheduledActivitiesV4(context);
```

Comment:
Should our unit test also verify that we return the expected activity, with the finishedOn timestamps and everything?